core/src/main/java/org/infinispan/transaction/impl/LocalTransaction.java

```diff
@@ -153,24 +152,6 @@ public String toString() {
             "} " + super.toString();
    }
 
-   @Override
-   public void addReadKey(Object key) {
-      CacheEntry entry = lookupEntry(key);
-      if (entry instanceof RepeatableReadEntry) {
-         ((RepeatableReadEntry) entry).setRead();
-      }
-   }
-
-   @Override
-   public boolean keyRead(Object key) {
-      CacheEntry entry = lookupEntry(key);
-      if (entry instanceof RepeatableReadEntry) {
-         return ((RepeatableReadEntry) entry).isRead();
-      } else {
-         return false;
```

Comment:
When possible, I prefer keeping deprecated methods working as before instead of keeping only a skeleton for compilation reasons.

Reply:
sorry I didn't understand what you mean :(

Reply:
I mean if the method still works, I don't see any reason to replace it with an empty method.